master/internal/trials/postgres_trials.go

```diff
@@ -70,8 +71,8 @@ func MetricsTimeSeries(trialID int32, startTime time.Time,
 		case db.MetricTypeBool:
 			cast = "boolean"
 		}
-		subq = subq.ColumnExpr("(metrics->?->>?)::? as ?",
-			metricsObjectName, metricName, bun.Safe(cast), bun.Ident(metricName))
+		subq = subq.ColumnExpr("(metrics->?->>?)::? as ?", metricsObjectName,
```

Comment:
the bug is just with ```bun.Ident``` and ```.``` right? Just wondering if there is any where else in the code base that might also have this

Reply:
Yes it's `bun.Ident`. We only use `bun.Ident` here and ExperimentIdFilter (which shouldn't be included for separate reasons; it is not supported by proto; see http://latest-main.determined.ai:8080/api/v1/experiments?experimentIdFilter={%22incl%22:[2841]} )